scripts/firmware.py

```diff
@@ -1598,45 +1619,43 @@ def check_dfu_util(self) -> bool:
             print("dfu-util is not installed. Please install it and try again.")
             return False
 
-    def dfu_loop(self) -> List[str]:
-        start_time = time.time()
-        timeout = 30  # Timeout in seconds
+    def stop_loop(self, detected_devices: List[str]) -> List[str]:
+        return detected_devices
```

Comment:
Why even have this? 😂

Reply:
It was early in the AM :D think i had it a different way initially and just didnt remove it haha